### ISSUE TITLE
Property if data is compressed

### ISSUE THREAD

**Issue by Vario** [mw99/DataCompression#19]
Is there a way to check if a given set of data is compressed in any way?
like data.isGzipped -> true or data.compression -> Compression.gzip || Compression.zip ,....

If not this would be a feature request

**Comment by mw99** [comment]
Yes that seems plausible. Will try to get this into the next release. Just some notes: This will not be possible for all compression methods. Probably only gzip and zip, because the other steams don't have an identifying header.

**Comment by Kimentanm** [comment]
```swift
extension Data {
    public var isGzipped :Bool {
        return self.starts(with: [0x1f,0x8b])
    }
}
```
You can use this code to check data is compressed with gzip
